spec/airport_spec.rb

```
@@ -0,0 +1,63 @@
+require 'airport'
+
+describe Airport do
+  describe "#land" do 
+    it 'lands a plane at the airport' do
+      plane = Plane.new
+      expect(subject.land(plane)).to eq plane
+    end
+   
+    it 'raises error when airport is full' do
+      Airport::DEFAULT_CAPACITY.times { subject.land(Plane.new) }
+      expect { subject.land(Plane.new) }.to raise_error 'Airport full'
+    end
+  end
+
+  describe "#take_off" do
+    it 'shows when a plane has left the airport' do
+      plane = Plane.new
+      airport = Airport.new
+      airport.land(plane)
+      expect(airport.take_off(plane)).to eq Airport::DEFAULT_TAKE_OFF_MESSAGE
+    end
+  end
+
+  describe 'initialization' do
+    it 'can override airport capacity' do
+      airport = Airport.new(Airport::DEFAULT_CAPACITY)
```

I'm not sure if this checks that the default capacity is actually being overridden.
The value you are over-riding with is the same as the default so might be hiding something